examples/file_based_server/server_config.json5

```diff
@@ -0,0 +1,176 @@
+// JSON 5 server configuration
+{
+  // Server Description
+  buildInfo: {
+    productUri: "build.product.test.org",
+    manufacturerName: "Manufacture GmbH",
+    productName: "Product Name",
+    softwareVersion: "0.1.0",
+    buildNumber: "0.9.4",
+    buildDate: "2022-11-02T15:31:03Z",
+  },
+  applicationDescription: {
+    applicationUri: "urn:open62541.server.application",
+    productUri: "urn:product.test.org",
+    applicationName: {
+      locale: "de-DE",
+      text: "Test text"
+    },
+    applicationType: 0,
+    gatewayServerUri: "GATEWAY",
+    discoveryProfileUri: "DISC",
+    discoveryUrls: [
+      "__open",
+      "_62541",
+      "opc ua"
+    ],
+  },
+  shutdownDelay: 5000.50,
+
+  verifyRequestTimestamp: 0,
+  allowEmptyVariables: 0,
+
+  // Networking
+  // Set up the local ServerUrls. They are used during startup to initialize the server sockets.
+  serverUrls: [
+    "opc.tcp://127.0.0.1:4840",
+  ],
+  tcpBufSize: 64000,
```

Comment:
Add an object `tcp` that regroups the following options.
And add a `tcpEnabled` boolean.

In the future we will also have websockets, http, and so on.
These should be individually configured/enabled.